packages/main/src/plugin/image-registry.ts

```diff
@@ -134,8 +137,8 @@ export class ImageRegistry {
     if (exists) {
       throw new Error(`Registry ${registryCreateOptions.serverUrl} already exists`);
     }
-    const registry = provider.create(registryCreateOptions);
-    this.telemetryService.track('createRegistry', {
+    const registry = await provider.create(registryCreateOptions);
```

Comment:
is that your IDE/tooling that is reporting the issue of missing await ?

Reply:
yep, as far as we now having credentials validation in `create` method, we have to wait, when credentials will be checked